homeassistant/components/tilt_ble/translations/ca.json

```diff
@@ -0,0 +1,21 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "El dispositiu ja est\u00e0 configurat",
```

Comment:
Please drop all the translations except en.json as these will be generated by lokalize

Reply:
Will do. Copied all from another BLE integration since the strings were the same, but I wasn't sure how that was supposed to work.

Thanks! Definitely no rush; enjoy the holiday!